pkg/cmd/flags/output.go

```diff
@@ -35,6 +36,7 @@ Examples:
   --output json                                            | output as json
   --output gotemplate=/path/to/my.tmpl                     | output as the provided go template
   --output out-file:/my/out --output log-file:/my/log      | output to /my/out and logs to /my/log
+  --output forward:127.0.0.1:24224:tracee                  | output via the Forward protocol to 127.0.0.1 with the tag 'tracee'
```

Comment:
The current parsing seems to be a two-element string so I've extended it to be 4 elements.
Not sure if instead we want an approach like: `--output forward-host:... --output forward-port`.

We also need to consider TLS and non-TCP connections potentially as well as any other auth support we might want. A more general approach to make this flexible and consistent might be required.

Reply:
@patrick-stephens wonder if we could make the settings fit in a URI? 

scheme:[user[:password]]@host:port[?params]#tag

eg:

`forward: tcp://user:password@host:port?timeout=10s#tracee`

what do you think?

Reply:
It's certainly a reasonable ask, I just see a few issues with it I want to flesh out:

- What if username or password contains special characters? `:` or `@`
- TLS is the other one - this feels like a more general thing for the whole agent so should it be a standalone flag, e.g. use TLS, do [not] verify TLS but use it, use this set of certs, etc.
- Ignore anything unknown or error? I personally say error as it is misconfiguration, e.g. you have a typo, but it impacts backwards/forwards compatibility of settings.

It might be better to make everything a set of params although we would have to (base64?) encode a password for special characters. We could include TLS options in the set of params though I think quite easily.

Is this how Aqua/Tracee normally does it? I'd like to align with whatever is the usual way - consistency is always best.

Reply:
Thinking about it, it would be good if we could just to standard `net/url` parsing of the whole option rather than have to cope with the split by colons so I will look at that. It means a larger change to the current configuration to cope though but then matches what you are after and is standard then via Golang.

Reply:
I've updated it now @josedonizetti to just use the URL parsing so it should follow what you suggested - although I did use a parameter for the tag too:

`forward:tcp://user:password@host:port?tag=mytag`

All parameters are optional with some defaults.